packages/app/src/hooks/useCustomizationData.test.ts

```diff
@@ -0,0 +1,190 @@
+import { renderHook, waitFor } from '@testing-library/react';
+import { useApi } from '@backstage/core-plugin-api';
+import { useCustomizationData } from './useCustomizationData';
+
+jest.mock('@backstage/core-plugin-api', () => ({
+  ...jest.requireActual('@backstage/core-plugin-api'),
+  useApi: jest.fn(),
+}));
+
+describe('useCustomizationData', () => {
+  const learningPathData = [
+    {
+      paths: 6,
+      minutes: 20,
+      description:
+        'Learn about k8s API fundamentals and the etcd operator, as well as how to use the Operator SDK with Go and Helm, review Ansible basics, and use Ansible to deploy Kubernetes modules.',
+      label: 'Building Operators on OpenShift',
+      url: 'https://developers.redhat.com/learn/openshift/operators',
+    },
+    {
+      paths: 3,
+      minutes: 20,
+      description:
+        'For the best experience in this learning path, we suggest that you complete the following learning resources in the order shown. When you click on a resource, it will open in a new tab. Keep this page open so you can easily move on to the next resource!',
```

Comment:
In the future, consider revising this text to the following: 

_For the best experience with this learning path, complete the following learning resources in the order in which they are listed. When you click on a resource, it will open in a new tab. Keep this page open so you can easily move on to the next resource._